server/resp/src/main/java/org/infinispan/server/resp/RespRequestHandler.java

```diff
@@ -19,8 +20,16 @@ public abstract class RespRequestHandler {
    protected final RespServer respServer;
 
    public static final AttributeKey<ByteBufPool> BYTE_BUF_POOL_ATTRIBUTE_KEY = AttributeKey.newInstance("buffer-pool");
+   private static final BiFunction<RespRequestHandler, Throwable, RespRequestHandler> HANDLE_FAILURE = (h, t) -> {
+      if (t != null) {
+         Resp3Handler.handleThrowable(h.allocator(), t);
+         // If exception, never use handler
+      }
+      return h;
```

Comment:
I think this may cause a NPE if an exception happened, no?

Reply:
I think we need to change this to `return this;` in the above if and make the lambda not static.

Reply:
Nice, updated.